libs/coin-modules/coin-sui/src/logic/broadcast.test.ts

```diff
@@ -0,0 +1,77 @@
+import { broadcast } from "./broadcast";
+import suiAPI from "../network";
+
+// Mock the suiAPI
+jest.mock("../network", () => ({
+  __esModule: true,
```

Comment:
any reason for adding this to the mock?

Reply:
for some reason, it doesn't work without it

Reply:
It is needed if the module has a `default` export, which is the case.